messaging/src/main/java/org/apache/falcon/messaging/JMSMessageConsumer.java

```diff
@@ -92,7 +92,8 @@ public void startSubscriber() throws FalconException {
 
             topicSession = (TopicSession) connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
             Topic destination = topicSession.createTopic(topicName);
-            topicSubscriber = topicSession.createDurableSubscriber(destination, FALCON_CLIENT_ID);
+            topicSubscriber = topicSession.createDurableSubscriber(destination, FALCON_CLIENT_ID,
```

Comment:
With this change, any message from post-processing will get filtered. We should clearly call it out as a backward-incompatible change/removed feature.

Also, if this change is in place, we can also remove some of the code that allows processing of messages from both post-processing action (which was done for backward compatibility) and Oozie JMS notification.